File: javascript/public/js/rental.js
"use strict";

var Rental = function (movie, daysRented) {
    this.movie = movie;
    this.daysRented = daysRented;
};

Rental.prototype.getMovie = function () {
    return this.movie;
};

Rental.prototype.getDaysRented = function () {
    return this.daysRented;
};

File: cpp/src/Rental.cpp
#include "Rental.h"

Rental::Rental(const Movie movie, int daysRented)
: movie(movie), daysRented(daysRented) {
}

Rental::~Rental() {
}

int Rental::getDaysRented(){
	return daysRented;
}

Movie Rental::getMovie(){
	return movie;
}
